src/scss/pages/profile.scss

```diff
@@ -11,4 +11,23 @@
     width: 200px;
 }
 
+}
+
+.profileCountries, .profilePreferences {
+    font-size: 0.8em;
+    text-align: center;
```

Comment:
I'd heavily argue against center aligning text. Maybe center align the element and left align the text for better readability.